tardis/transport/montecarlo/packet_trackers.py

```diff
@@ -14,6 +18,9 @@
     ("energy", float64[:]),
     ("shell_id", int64[:]),
     ("interaction_type", int64[:]),
+    ("in_nu", float64[:]),
+    ("in_id", int64[:]),
```

Comment:
rename to `line_in_id`

Reply:
`interaction_in_line_id`

Reply:
line_interaction_in_id
line_interaction_out_id
line_interaction_in_nu
???